apps/camera_pipe/camera_pipe.cpp

```diff
@@ -15,13 +17,18 @@ Expr avg(Expr a, Expr b) {
 }
 
 Func hot_pixel_suppression(Func input) {
+    // hot pixel maximum
     Expr a = max(max(input(x-2, y), input(x+2, y)),
                  max(input(x, y-2), input(x, y+2)));
-    Expr b = min(min(input(x-2, y), input(x+2, y)),
-                 min(input(x, y-2), input(x, y+2)));
+    // cold pixel minimum
+    // Expr b = min(min(input(x-2, y), input(x+2, y)),
```

Comment:
@jrk @abadams to comment on that. In general, I don't really understand why this would be the case, I'll look at the images later with/without this change to see if I can get a better idea of what is going on. Maybe there's a bug related to this.